package.json

```diff
@@ -41,8 +41,10 @@
     "@types/require-directory": "^2.1.2",
     "@types/serve-favicon": "^2.5.4",
     "applicationinsights": "^3.2.1",
+    "archiver": "^7.0.1",
```

Comment:
For the commands e.g. test:routes do we need these?

Can you remove what we don't need please

Reply:
removed stuff